ERROR - 2022-11-18 18:09:16 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#9 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-18 18:09:16 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Products->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-18 18:10:47 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#9 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-18 18:10:47 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Products->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-18 18:11:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#9 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-18 18:11:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Products->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-18 18:12:03 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#9 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-18 18:12:03 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Products->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-18 18:13:02 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#9 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-18 18:13:02 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Products->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-11-18 18:14:10 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#9 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\xampp\htdocs\CodeIgniter4\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\CodeIgniter4\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\CodeIgniter4\system\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\CodeIgniter4\system\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\CodeIgniter4\app\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(927): App\Controllers\Products->index()
#7 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
#8 C:\xampp\htdocs\CodeIgniter4\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\CodeIgniter4\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2022-11-18 18:14:10 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `product`', [], false)
 3 SYSTEMPATH\Model.php(240): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(586): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Products.php(14): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\Products->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Products))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
